component/rbac.libsonnet

```diff
@@ -56,10 +56,29 @@ local clusterRoleBindingAdmin = kube.ClusterRoleBinding('impersonate-' + params.
   },
 };
 
+local sudoAlertmanagerAccess =
+  kube.RoleBinding('alertmanager-access-' + kube.hyphenate(params.sudoGroupName)) {
+    metadata+: {
+      namespace: 'openshift-monitoring',
+    },
+    subjects: [ {
+      apiGroup: 'rbac.authorization.k8s.io',
+      kind: 'Group',
+      name: params.sudoGroupName,
+    } ],
+    roleRef_: {
+      kind: 'Role',
+      metadata: {
+        name: 'monitoring-alertmanager-edit',
```

Comment:
As far as I can see this role doesn't exist on OCP < 4.9, but having a RoleBinding to an nonexistent role shouldn't be an issue.

Reply:
I agree, I don't think it's a problem